tests/dummy/app/router.js

```diff
@@ -1,12 +1,13 @@
 import Ember from 'ember'
+const {Router} = Ember
```

Comment:
You can destructure `Ember.Router()` and create an alias `EmberRouter` like this.